README.md

```diff
@@ -1 +1,24 @@
-# hocs-bulk-case-action
\ No newline at end of file
+# Hocs-Bulk-Case-Action
+
+This project is intended to be used when one off bulk operations need to be
+performed on live cases.
```

Comment:
This doesn't actually explain to me what this repo is?

Reply:
It's in the other repos, but given that I don't think anyone has even managed to get DECS running on Windows very successfully I agree it's just clutter so I'll get rid